common-npm-packages/packaging-common/universal/ArtifactToolUtilities.ts

```diff
@@ -175,5 +175,5 @@ export async function getHighestPackageVersionFromFeed(serviceUri: string, acces
         }
     }
     
-    return null;
+    return "0.0.0";
```

Comment:
This function is only used by the universalpackagesV0 task. As the function name implies, the highestPackageVersion should be returned. In the event no package yet exists, a (0.0.0) should be returned.